transport_nantes/topicblog/models.py

```diff
@@ -776,9 +776,21 @@ class TopicBlogEmailSendRecord(models.Model):
 
     """Represent the fact that we sent an email.
     """
+
+    class StatusChoices(models.TextChoices):
+        # This is the default state
+        PENDING = 'PENDING', "Pending"
+        # Toggled to this state when Amazon SES sends the email
+        SENT = 'SENT', "Sent"
```

Comment:
Present tense makes more sense.
More importantly, it's when SES _accepts_ the mail for sending.

Reply:
Changed to 
`# Toggle to this state when Amazon SES accepts to send the email`